Framework/PythonInterface/test/python/plugins/algorithms/VesuvioCorrectionsTest.py

```diff
@@ -27,6 +31,15 @@ def setUp(self):
         if self._test_container_ws is None:
             self.__class__._test_container_ws = create_test_container_ws()
 
+        self._input_bins = self._test_ws.blocksize()
+
+        # Check OS for RHEL7 or Ubuntu
+        dist = platform.linux_distribution()
```

Comment:
I see. I agree I think that is better. Can we rename `is_linux` to `is_rhel6`? There are a lot of Linux distributions and I think we need to be clear we're being very specific here.